extensions/azurecore/src/azureResource/providers/database/databaseTreeDataProvider.ts

```diff
@@ -42,7 +42,7 @@ export class AzureResourceDatabaseTreeDataProvider extends ResourceTreeDataProvi
 				databaseName: database.name,
 				userName: database.loginName,
 				password: '',
-				authenticationType: 'SqlLogin',
```

Comment:
Remove the hardcoded SqlLogin option from the Azure Resource Tree pane.  Later, we'll set this value from user configuration.  Also, we'll have a sensible default value in code if any issues.